modules/qtwidgets/src/properties/lightpropertywidgetqt.cpp

```diff
@@ -73,20 +123,29 @@ LightPropertyWidgetQt::LightPropertyWidgetQt(FloatVec3Property* property)
         LogWarn("Maximum value is assumed to be positive. Widget may produce values out of range.")
     }
 
-    vec3 maxVal = glm::abs(property_->getMaxValue());
-    radiusSpinBox_->setMinimum(-glm::length(maxVal));
-    radiusSpinBox_->setMaximum(glm::length(maxVal));
+    radius_->setMinValue(0, ConstraintBehavior::Immutable);
```

Comment:
immutable?

Reply:
it's a length can never be less then 0, I've removed the hack with negative distance from the gui at least